controllers/inventory-controller.js

```diff
@@ -69,7 +68,77 @@ const getItemById = async (req, res) => {
   }
 };
 
+//POST a new inventory item
+const postNewInventoryItem = async (req, res) => {
+  try {
+    const {
+      item_name,
+      description,
+      category,
+      status,
+      quantity,
+      warehouse_name,
+    } = req.body;
+
+    // Validate data
```

Comment:
for best practice, consider moving definitions of functions like validation or warehouseID look up outside of getItemById (possibly inside a helper file) for code reusability and not distracting the main purpose of getItemById